source/fundamentals/builders/aggregates.txt

```diff
@@ -19,7 +19,10 @@ In this guide, you can learn how to use the `Aggregates <{+api+}/apidocs/mongodb
 class which provides static factory methods that build :manual:`aggregation pipeline
 stages </meta/aggregation-quick-reference/#stages>` in the MongoDB Java driver.
 
-For a more thorough introduction to Aggregation, see our :ref:`Aggregation guide <java-aggregation>`.
+These builders serve as a wrapper for aggregation expressions and provide some
+aggregation functionality. For a more thorough introduction to Aggregation, see
```

Comment:
Question:

I could be remembering incorrectly, but didn't Maxim want to avoid any mention of these expressions as a "wrapper"? If so, could the first sentence be omitted?